assets/css/styles.css

```diff
@@ -721,14 +738,291 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
   }
 }
 
+/*
+  ADD PROJECT POPUP CSS
+  - Add the Popup class (same as the html file and style it)
+  --  Add .popup button and .popup h1, h2, h3 and p to respective
+      class declarations in the different sections
+  - Add the class open-popup (to be added in the javascript open function)
+*/
+.modal-container {
+  width: 100vw;
+  height: 100vh;
+  position: relative;
+  top: 0;
+}
+
+.popup-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background: var(--my-competences-boxes-color);
+  z-index: 1;
+  filter: opacity(95%);
+  overflow-y: hidden;
+  overscroll-behavior: none;
+}
+
+.project-popup {
+  width: 100%;
+  position: absolute;
+  background: var(--image-placeholder-color1);
+  border-radius: 1%;
+  top: 0;
+  left: 0;
+  text-align: center;
+  padding: 0;
+  transition: transform 0.4s, visibility 0.4s, top 0.4s;
+  visibility: visible;
+  z-index: 100;
+  backdrop-filter: blur(15px);
+  backdrop-filter: brightness(60%);
+  backdrop-filter: contrast(40%);
+}
```

Comment:
- [ ] [Optional] Please stick to the design as much as possible. Please make your mobile design a Figma design.
- [ ] [Optional] Please stick to the design like color, buttons and `X` buttons, spacing etc.
### Your design
![his](https://user-images.githubusercontent.com/81727988/233697029-d01d5eb4-25c8-456b-8057-b2b435f4ca8f.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/233697054-16f5c070-572a-4522-9c18-5a8078cf6ed4.png)